SimPEG/PF/Magnetics.py

```diff
@@ -571,35 +694,39 @@ def Jvec(self, m, v, u=None):
             u = self.fields(m)
 
         B, u = u['B'], u['u']
-        mu = self.muMap*(m)
+        mu = self.muMap * (m)
         dmudm = self.muDeriv
-        dchidmu = Utils.sdiag(1/mu_0*np.ones(self.mesh.nC))
+        # dchidmu = sdiag(1 / mu_0 * np.ones(self.mesh.nC))
 
         vol = self.mesh.vol
         Div = self._Div
         Dface = self.mesh.faceDiv
         P = self.survey.projectFieldsDeriv(B)  # Projection matrix
         B0 = self.getB0()
 
-        MfMuIvec = 1/self.MfMui.diagonal()
-        dMfMuI = Utils.sdiag(MfMuIvec**2)*self.mesh.aveF2CC.T*Utils.sdiag(vol*1./mu**2)
+        MfMuIvec = 1 / self.MfMui.diagonal()
+        dMfMuI = sdiag(MfMuIvec**2) * \
```

Comment:
for splitting math over lines, I find using brackets much easier to read than `\` e.g. 
```
dMfMuI = (
    sdiag(MfMuIvec**2) *
    self.mesh.aveF2CC.T ... 
)
```